evals/scripts/create_tiktok_vi_comments_registry_data.py

```diff
@@ -6,18 +6,50 @@
 from consts import api_endpoint
 from get_instruction import get_instruction
 
+def fetch_default_dataset():
+    encoded_postURLs = urllib.parse.quote("[\"https://www.tiktok.com/@bellapoarch/video/6862153058223197445\"]")
+    url = f"{api_endpoint}/api/get_tiktok_video_comments?postURLs={encoded_postURLs}"
+    response = requests.get(url)
+    data = response.json()
+    defaultDatasetId = data['data']
+
+    return defaultDatasetId
+
+def fetch_comments(datasetId):
+    response = requests.get(f"{api_endpoint}/api/get_tiktok_video_comments/get_dataset_items?datasetId={datasetId}")
+    data = response.json()
+    if len(data['data']['videos']) > 0:
+        return data['data']
+    else:
+        return None
+
 def create_tiktok_vi_comments_registry_data(ideal_key_or_value, question):
     registry_data = ""
+    defaultDatasetId = fetch_default_dataset()
```

Comment:
For the evals, could you please set the `comments` to `3` to keep our costs low?
- actual: 100 comments
- expected: 3 comments

Reply:
Alternatively, hard-code the `datasetId` so we do not query for new comments during evals to keep API costs low.